frontend/src/pages/LoginPage/LoginPage.tsx

```diff
@@ -64,46 +71,90 @@ const LoginPage = () => {
     }
 }
 
-  return (
-    <>
-      <div className="container">
-        <div className="form-container">
-          <h2 className="title">Login</h2>
-          <Form onSubmit={handleLoginSubmit}>
-            <Form.Group className="mb-3 form-group" controlId="formBasicEmail">
-              <Form.Label className="form-label">Email address</Form.Label>
-              <Form.Control
-                type="email"
-                placeholder="Enter email"
-                className="form-control"
-                value={loginFormData.email}
-                onChange={(e: React.ChangeEvent<HTMLInputElement>) => handleInputChange('email', e.target.value)}
-              />
-            </Form.Group>
-            <Form.Group className="mb-3 form-group" controlId="formBasicPassword">
-              <Form.Label className="form-label">Password</Form.Label>
-              <Form.Control
-                type="password"
-                placeholder="Password"
-                className="form-control"
-                value={loginFormData.password}
-                onChange={(e: React.ChangeEvent<HTMLInputElement>) => handleInputChange('password', e.target.value)}
-              />
-            </Form.Group>
-            <Button variant="primary" type="submit" className="btn-primary">
-              Submit
+return (
+  <>
+  <ThemeProvider theme={defaultTheme}>
```

Comment:
Delete the ThemeProvider since it's useless here. No theme was actually provided.